backend/usb-libusb.c

```diff
@@ -1722,7 +1722,7 @@ static void *read_thread(void *reference)
     */
 
     if ((g.wait_eof || !g.read_thread_stop))
-      usleep(250000);
+      usleep(2000000);
```

Comment:
Might as well make this "sleep(2)" for clarity...

Reply:
Updated to use sleep(2) and the comment was updated too.